### PR TITLE
Support user black list

### LINKED ISSUE #250: Don't analyze wptsync patches
They already submit them to try to make sure that everything is OK, so it's not worth for us to analyze them again (especially since they are huge stacks and we submit one push per patch in the stack!).

### PR DESCRIPTION
**Pull request by La0**
Fixes #250

### CODE REVIEW

File: events/code_review_events/workflow.py [hunk 1]
```diff
@@ -134,6 +153,23 @@ def get_repositories(self, repositories, cache_root):
                 # Requeue when nothing changed for now
                 await self.bus.send(QUEUE_WEB_BUILDS, build)
 
+    def is_blacklisted(self, revision: dict):
+        """Check if the revision author is in blacklisted"""
+        author_phid = revision["fields"]["authorPHID"]
+        author = next(
```
Comment (marco-c) [inline events/code_review_events/workflow.py]: It'd be nicer to make self.user_blacklist a set of PHIDs, so you can simplify this

File: events/code_review_events/workflow.py [hunk 2]
```diff
@@ -247,14 +283,14 @@ def should_run_test_selection(self, build):
         try:
             if self.should_run_test_selection(build):
                 task = self.community_hooks.triggerHook(
-                    "project-relman", "bugbug-test-select", {"DIFF_ID": build.diff_id},
+                    "project-relman", "bugbug-test-select", {"DIFF_ID": build.diff_id}
```
Comment (marco-c) [inline events/code_review_events/workflow.py]: I wonder why this was changed now by pre-commit :S

File: events/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,5 @@
 -e ../tools #egg=code-review-tools
 libmozevent==1.0.5
+
+# Use updated libmozdata with search_users
+https://github.com/mozilla/libmozdata/archive/9923f1a6e8aa20a6f042f8808a380eb0f5c87b49.zip #egg=libmozdata
```
Comment (marco-c) [inline events/requirements.txt]: We should just release a new version

Comment (La0) [reply events/requirements.txt]: Done: https://pypi.org/project/libmozdata/0.1.63/

File: events/code_review_events/workflow.py (hunk 2 above)
Comment (La0) [reply events/code_review_events/workflow.py]: :bomb:

File: events/code_review_events/workflow.py (hunk 1 above)
Comment (La0) [reply events/code_review_events/workflow.py]: Yes but i'll lose the naming

**Pull request merged**